doc/hands_on/hands_on.md

```diff
@@ -0,0 +1,194 @@
+# Hands on S3-Virtual-Schema
+
+![Hands on S3-Virtual-Schema](./title.png)
+
+## Introduction
+
+In this guide we will walk through the setup of the Virtual Schema for document files stored on [AWS S3](https://aws.amazon.com/s3/).
+
+You need:
+
+* An Exasol database
+* An AWS account
+
+## What is This for?
+
+In this tutorial we will create a virtual schema for JSON-documents that we store in a S3 Bucket.
+If you've never heard of S3, that no problem. You can imagine it as a very basic file storage.
+You can store files and access them using a key.
+
+We are going to create a Virtual Schema for these files, so that you can access the JSON data just like a regular Exasol tabel.
+
+So you may ask yourself "Why not just create an Exasol table and import the data?".
+While in same cases this might even be the better solution, but there are also use cases where you still want to create a Virtual Schema:
+
+### S3 as Cold-Storage
+
+The first use case is that you have lots of data that you don't need very often in you analytics.
+In that case you can use S3 as a cheap cold storage and integrate the data only when you needed them into your analytics.
+In case you can identify the data you need by the file name, you can use a filter on the file name, and the Virtual Schema adapter will only transfer the single file.
+
+### Use it for Import
```

Comment:
To be consistent start the verbs with `-ing` ending.
Check also other headings.